blog/entries/templates/entries/tag_form.html

```diff
@@ -0,0 +1,19 @@
+{% extends "entries/base.html" %}
```

Comment:
this is a default convention django createview uses to figure out what form to use for a given model